src/camlsnark_c/libsnark-caml/depends/ate-pairing/include/cybozu/benchmark.hpp

```diff
@@ -5,7 +5,7 @@
 	@author MITSUNARI Shigeo
 */
 #if defined(_MSC_VER) && (MSC_VER <= 1500)
-	#include <cybozu/inttype.hpp>
+	#include "cybozu/inttype.hpp"
```

Comment:
I oppose this change, see my comment [here](https://github.com/o1-labs/snarky/pull/524#issuecomment-681968731) for more context.

Reply:
No prob, will fix.